api/send_email.py

```diff
@@ -0,0 +1,19 @@
+# Third-Party Imports
+from decouple import config
+from django_q.tasks import async_task
+
+# App Imports
+from core.constants import MSG, SUBJECT
+from core.models.user import User
+
+
+def send_email(data):
+    """"This method sends emails to  users"""
+    for user in User.objects.filter(is_staff=True):
+        async_task(
+            "django.core.mail.send_mail",
```

Comment:
Consider using [send_messages](https://docs.djangoproject.com/en/2.2/topics/email/#sending-multiple-emails) for sending mails. It's more efficient.